adr/003-initial-datamodel.md

```diff
@@ -0,0 +1,32 @@
+# 2. Initial data model
+
+Date: 2019-09-24
+
+## Status
+
+Accepted
+
+## Context
+
+In the coming months we will be building this application. The first order of business is to come up with a sensible data model and application structure.
+
+## Decision
+
+We discussed a database model that is based around a small number of tables with a lot of columns. In particular:
+
+- `Candidate`, primarily used for authentication
+- `ApplicationForm`, which belongs to a Candidate, and holds the
+information that is common to all of the applications that the user
+makes (in the "Apply 1" phase, they can apply for up to three courses).
+- `CourseChoice`, which holds the information that is course specific,
```

Comment:
I have some reservations about this class name.

Based on the name, I erroneously assumed that this model would model the choice of the course (e.g. the course and the training location) – I've even independently used this name [on the prototype branch](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/208).

Reply:
I've put a quick meeting in at 3:30 this afternoon for us to discuss this